modules/apps/frontend-theme/frontend-theme-classic/src/templates/navigation.ftl

```diff
@@ -3,7 +3,7 @@
 		<span class="navbar-toggler-icon"></span>
 	</button>
 
-	<div aria-expanded="false" class="collapse navbar-collapse" id="navigationCollapse">
```

Comment:
Isn't this a collapsible menu on small screen sizes?

Reply:
Hh, yeah, but the element which needs the aria-expanded is the trigger, right now we have duplication in the trigger and in the container:

https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-theme/frontend-theme-classic/src/templates/navigation.ftl#L2

The relation between trigger an element hidden is done by the `aria-controls="ID"`

An example in bootstrap here:
https://getbootstrap.com/docs/4.5/components/navbar/#supported-content

Reply:
fair enough, I just checked the diff here 👍